tests/test_lint_recipe.py

```diff
@@ -817,6 +817,40 @@ def test_string_source(self):
         ).format(type(url).__module__, type(url).__name__)
         self.assertIn(msg, lints)
 
+    def test_single_space_pins(self):
+        meta = {
+            "requirements": {
+                "build": [
+                    "{{ compilers('c') }}",
+                    "python >=3",
+                    "pip   19",
+                ],
+                "host": [
+                    "python >= 2",
+                ],
+                "run": [
+                    "xonsh>1.0",
+                    "conda= 4.*",
+                    "conda-smithy<=54.*",
+                ],
+            }
+        }
+        lints, hints = linter.lintify(meta)
+        filtered_lints = [lint for lint in lints if lint.startswith("``requirements: ")]
+        expected_messages = [
+            "``requirements: build: pip   19`` should only contain a single space "
+            "between the name and the pin, i.e. ``pip 19``",
+            "``requirements: host: python >= 2`` should only contain a single space "
+            "between the name and the pin, i.e. ``python >=2``",
```

Comment:
This message is confusing. It should be something like no space between relational operator and version

Reply:
Feel free to put in a PR with better language!